pkg/promtail/targets/filetarget.go

```diff
@@ -16,7 +18,7 @@ import (
 	fsnotify "gopkg.in/fsnotify.v1"
 
 	"github.com/grafana/loki/pkg/helpers"
-	"github.com/grafana/loki/pkg/promtail/api"
```

Comment:
Super nit picking: go imports tend to be order; stdlib, external package, internal package.